omod/src/main/webapp/resources/js/ugandaemr.js

```diff
@@ -839,3 +839,23 @@ function getLatestEncounterFromEncounterList(encounterList) {
     return latestEncounter
 }
 
+/**
+ *
+ * function returns the name of the Encounter type Uuid passed
+ * @param encounterType Uuid
+ * @returns {string} Naame of Encounters
+ */
+function getEncounterTypeName(encounterType) {
+    var encounterTypeName="";
+    jq.ajax({
+        type: "GET",
+        url: '/' + OPENMRS_CONTEXT_PATH + "/ws/rest/v1/encountertype/"+encounterType+"?v=full",
```

Comment:
@dbaluku I think here you do not need to add v=full, you can add custom:name to just return the name and no other attributes which are not needed, better for performance

Reply:
Done